PackageViewModel/PackageViewModel.cs

```diff
@@ -235,6 +239,21 @@ public ICollection<PackagePart> PackageParts
             }
         }
 
+        public List<object?> SelectedItems
+        {
+            get { return _selectedItems; }
+            set
+            {
+                if (!_selectedItems.Contains(value))
```

Comment:
Sorry about not beeing clear enough the first time but this line also looks  a bit strange :)
And as mentioned before you might be able to just get rid of the setter because in my short testing it never got called.

Reply:
Tested it out without having setter and works fine. My bad not seeing this. Fixed :)